<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>avlform</class>
 <widget class="QDialog" name="avlform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(5,2,13);</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>10</y>
     <width>901</width>
     <height>681</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>899</width>
      <height>679</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="labelGrafico_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>915</x>
     <y>10</y>
     <width>381</width>
     <height>681</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #9DAABC;
</string>
   </property>
   <widget class="QPushButton" name="btnInsertar">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>530</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(28,19,42);
    border-style: outset;
    border-width: 0px;
    border-radius: 8px;
    border-color: beige;
    font: bold 18px;
	color: rgb(254,215,215);
    padding: 6px;
}</string>
    </property>
    <property name="text">
     <string>Insertar</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="inputT">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>530</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QTextEdit{
border:2px;
border-style:solid;
border-color:rgb(28,19,42);
border-radius: 8px;
}

QTextEdit:focus {
    background-color: #DDD;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBuscar">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>580</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(28,19,42);
    border-style: outset;
    border-width: 0px;
    border-radius: 8px;
    border-color: beige;
    font: bold 18px;
	color: rgb(254,215,215);
    padding: 6px;
}</string>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
font: 14pt &quot;Cooper Black&quot;;
color: black;</string>
    </property>
    <property name="text">
     <string>Estructura del Nodo</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelCodeImg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>361</width>
      <height>331</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>381</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(5,2,13);</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
font: 25pt &quot;Cooper Black&quot;;
color: white;</string>
     </property>
     <property name="text">
      <string>AVL Tree</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="inputTFind">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>580</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QTextEdit{
border:2px;
border-style:solid;
border-color:rgb(28,19,42);
border-radius: 8px;
}

QTextEdit:focus {
    background-color: #DDD;
}</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
